vendor/github.com/go-playground/locales/currency/currency.go

```diff
@@ -0,0 +1,308 @@
+package currency
+
+// Type is the currency type associated with the locales currency enum
+type Type int
```

Comment:
we could remove the vendor directory and add it to `.gitignore` since we've the dependencies with dep. This could go as a separate PR/issue. and others could add more if there's any reason for those checkedin dependencies.

Reply:
We are currently committing the vendor directory, this prevents upstream VCS changes from breaking builds.